src/main/resources/config/en.yml

```
@@ -83,6 +83,9 @@ PositionPlace:
 
 RotationPlace:
   cancelvl: 5
+  flagBufferMax: 1 # set to -1 to disable
+  flagBufferIncrement: 1
+  flagBufferDecay: 0.1
 
```

I feel like this would be better for a config?
```suggestion
  flag-buffer-max: 1 # set to -1 to disable
  flag-buffer-increment: 1
  flag-buffer-decay: 0.1

```